src/errors/Messages.js

```diff
@@ -81,6 +81,8 @@ const Messages = {
 
   PRUNE_DAYS_TYPE: 'Days must be a number',
 
+  GUILD_CHANNEL_INVITE: 'Could not find a available channel to create a invite.',
```

Comment:
Is my english rusty? I don't know if this is correct or not, so booping @iCrawl 

```suggestion
  GUILD_CHANNEL_INVITE: 'Could not find an available channel to create a invite.',
```

Reply:
> create an invite

apart from that looks fine